engine/src/main/java/com/datatorrent/stram/cli/ApexCli.java

```diff
@@ -2996,6 +2997,21 @@ private File copyToLocal(String[] files) throws IOException
     return tmpDir;
   }
 
+  public static class GetAppPackageInfoCommandLineOptions
+  {
+    final Options options = new Options();
+    final Option withDescription = add(new Option("withDescription", false, "Get default properties with description"));
+
+    private Option add(Option opt)
+    {
+      this.options.addOption(opt);
+      return opt;
+    }
+
+  }
+
+  private static GetAppPackageInfoCommandLineOptions GET_APP_PACKAGE_INFO_OPTIONS = new GetAppPackageInfoCommandLineOptions();
```

Comment:
Change from GetAppPackageInfoCommandLineOptions to Options and add "withDescription" option in a static block.